src/handlers/payout/post.ts

```diff
@@ -57,7 +57,7 @@ export const incentivizeComments = async () => {
     const rewardValue = calculateRewardValue(commentsByNode, commentElementPricing);
     logger.debug(`Comment parsed for the user: ${user}. comments: ${JSON.stringify(commentsByNode)}, sum: ${rewardValue}`);
     const account = await getWalletAddress(user);
-    const amountInETH = ((rewardValue * baseMultiplier) / 1000).toString();
```

Comment:
https://github.com/ubiquity/ubiquibot/blob/866dc9369a8e75297d5519cd42b6617086cf73c8/src/handlers/payout/post.ts#L135
This is the calculation for issue creator reward, so you should remove the division there.

Or perhaps for both issue creator reward and comments reward? @pavlovcik

Reply:
It's actually unclear to me right now perhaps I've been working too long today and will need to check the code again with fresh eyes. 

It will help me if you can explain to me without looking at the code now though.

Reply:
He changed the formula for comment reward, but not for issue creator reward, which this issue is about.

Reply:
@whilefoo I found the original PR here https://github.com/ubiquity/ubiquibot/pull/382/files which introduced calculation in generatePermitForComments , I will push the update now.

Reply:
@whilefoo @pavlovcik I amended the commit , will still wait for QA (@Draeieg ) to confirm that it works as expected.

Reply:
> He changed the formula for comment reward, but not for issue creator reward, which this issue is about.

I understand now. We should change all the calculations so that in the config we don't need to enter in a number like `1000` when we can enter in `1`